fly.toml

```diff
@@ -12,6 +12,9 @@ auto_rollback = true
 source = "data"
 destination = "/data"
 
+[deploy]
+release_command = "npm run upload:sentry"
```

Comment:
I think I'd rather move this command to the Dockerfile.

Reply:
@kentcdodds Unfortunately, I don't think this is possible. The environment variables from Fly.io are not available during build time, it has to come post-build it seems:

https://community.fly.io/t/reading-env-variables-during-deployment/12935/2

Reply:
Could we change this to `node ./scripts/sentry-create-release` so we don't need the npm script?